configs/mender_convert_config

```diff
@@ -230,6 +230,7 @@ mender_create_artifact() {
   local -r device_type="${1}"
```

Comment:
Idea:
Perhaps `device_type` is now better named as `device_types` ? As really, this is what it is now, and device_type is just a device_types instance with one type.

Reply:
Yes, please. Otherwise is very confusing.

Reply:
Will do :+1: